lib/injection/managers_injection.dart

```diff
@@ -1,18 +1,19 @@
 import 'package:flutter/foundation.dart';
 
 import 'package:api/dependency_injection.dart';
-import 'package:api/models/app/managers/connection_status_manager.dart';
-import 'package:api/models/app/managers/remote_config.dart';
-import 'package:api/models/app/managers/crash.dart';
 
+import 'package:globo_fitness/managers/connection_status_manager_impl.dart';
 import 'package:globo_fitness/managers/connection_check_status_manager.dart';
 
 import 'package:data/firebase/remote_config/firebase_remote_config.dart';
 import 'package:data/firebase/crashlytics/firebase_crashlytics.dart';
 
+import 'package:api/models/app/managers/remote_config.dart';
+import 'package:api/models/app/managers/crash.dart';
+
 Future<void> registerManagers() async {
   DependecyInjection.instance
-      .inject<ConnectionStatusManager>(ConnectionCheckStatusManager());
+      .inject<ConnectionStatusManagerImpl>(ConnectionCheckStatusManager());
```

Comment:
should it not be the .inject<ConnectionStatusManager>(ConnectionCheckStatusManagerImp()); ?